plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -273,21 +280,55 @@ func (m *Win_PerfCounters) AddItem(counterPath string, objectName string, instan
 			return err
 		}
 
+		origObjectName, _, origCounterName, err := extractCounterInfoFromCounterPath(origCounterPath)
+		if err != nil {
+			return err
+		}
+
 		for _, counterPath := range counters {
 			var err error
-			counterHandle, err := m.query.AddCounterToQuery(counterPath)
 
 			objectName, instance, counterName, err = extractCounterInfoFromCounterPath(counterPath)
 			if err != nil {
 				return err
 			}
 
+			var newItem *counter
+			if !m.LocalizeWildcardsExpansion {
+				// On localized installations of Windows, Telegraf
+				// should return English metrics, but
+				// ExpandWildCardPath returns localized counters. Undo
+				// that by using the original object and counter
+				// names, along with the expanded instance.
+
+				var newInstance string
+				if instance == "" {
+					newInstance = emptyInstance
+				} else {
+					newInstance = instance
+				}
+				counterPath = formatPath(origObjectName, newInstance, origCounterName)
```

Comment:
I think you duplicate the decision for an empty instance here. How about directly determining the `counterPath` by
```suggestion
				if instance == "" {
					counterPath = "\\" + origObjectName + "\\" + origCounterName
				} else {
					counterPath = "\\" + origObjectName + "(" + instance + ")\\" + origCounterName
				}

```

Or pass the `instance` to `formatPath()` directly and add the empty value to the if/else case there.

Reply:
This section is definitely a little awkward. I was trying to use formatPath to format the "unlocalized" path, but formatPath expects the '------' placeholder that the config file allows. Since counterPath here didn't come from the config file, but from `GetCounterPath(counterHandle)` which is a windows API and doesn't know about the placeholder, I had to add the placeholder for formatPath to work.

There is definitely an opportunity to clarify the code here but I'd like to leave it as is because I've manually tested it. Let's put off cleaning it up until we have automated coverage for this case.